notebooks/04_model_select_and_optimize/azureml_hyperdrive_surprise_svd.ipynb

```diff
@@ -203,7 +203,7 @@
    "source": [
```

Comment:
<p>We defined `USERCOL` and `ITEMCOL` earlier. We can use them for:</p><p>```</p><p>compute_rating_predictions(svd, test, usercol="userID", itemcol="itemID")</p><p>```</p><p>and</p><p>```</p><p>compute_ranking_predictions(svd, train, usercol="userID", itemcol="itemID", remove_seen=REMOVE_SEEN)</p><p>```</p><p><br></p><p>Btw, we don't pass the user and item column names to evaluation functions so that they use the default names., e.g.</p><p>```</p><p>eval(metric)(test, all_predictions, col_prediction='prediction', k=K)</p><p>```</p><p><br></p><p>I think we should either use all default or set all.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/840/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>